src/library_webgpu.js

```diff
@@ -146,6 +146,14 @@ wgpu${type}Release: (id) => WebGPU.mgr${type}.release(id),`;
       Instance: 1,
       VertexBufferNotUsed: 2,
     },
+    CreatePipelineAsyncStatus: {
```

Comment:
nit: Sort this alphabetically in the list (between BufferMapAsyncStatus and ErrorType)

Reply:
Done in 84dae207